### ISSUE TITLE
Keyboard tooltip of item in toolStrip2 is not displayed after expanded and collapsed the toolStripDropDownbutton in toolStrip1, then tab to toolStrip2's item

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#12818]
### .NET version

10.0.100-alpha.1.25064.17

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

Repro in previous .NET versions

### Issue description

When expand and collapse the toolStripDropDownbutton in toolStrip1, then tab to toolStrip2's item, cannot see keyboard tooltip:
Also repro between these controls: MenuStrip/ToolStrip/StatusStrip

![Image](https://github.com/user-attachments/assets/b6c24b72-b3aa-49a6-a269-f2e4324b5514)

https://github.com/user-attachments/assets/81d5b97d-b10e-4579-9690-c841e4768f83

### Steps to reproduce

1.	Create Winforms .NET app with 2 toolStrips
2.	In these 2 toolStrips, add ToolStripDropDownButton, add toolStripMenuItem for ToolStripDropDownButton separately
3.	Set these 2 toolStrips' TabStop property to True
4.	Build and run app
5.	Press down to expand ToolStripDropDownButton in toolStrip1 - it will focus on toolStripMenuItem
6.	Press Alt key to collapse ToolStripDropDownButton in toolStrip1, tab to toolstrip2 button

[WinFormsApp81.zip](https://github.com/user-attachments/files/18488597/WinFormsApp81.zip)

**Comment by Olina-Zhang** [comment]
> [@Olina-Zhang](https://github.com/Olina-Zhang) - does this work on .NET Framework?

It doesn't work in .NET framework, have same behavior.

**Comment by Zheng-Li01** [comment]
Verified the issue on latest .NET SDK 10.0.100-preview.2.25112.47 + private dlls built from  the main branch of Winforms repo. the issue has been fixed that the Keyboard tooltip display well as below screenshot.
![Image](https://github.com/user-attachments/assets/e3f905ff-65d2-4f6c-9c62-335838ce0be8)

**Comment by chaowendy** [comment]
Verified with .NET SDK 10.0 P2 test pass build, it was fixed as above testing result.
